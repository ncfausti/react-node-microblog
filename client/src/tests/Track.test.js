/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import Track from '../components/Track';

test('Page matches snapshot', () => {
  const component = renderer.create(
    <Track track={null} />,
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
