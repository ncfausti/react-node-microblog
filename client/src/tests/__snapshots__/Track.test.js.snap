// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page matches snapshot 1`] = `
<div
  className="track"
/>
`;
